website/docs/r/resource_pool.html.markdown

```diff
@@ -0,0 +1,116 @@
+  of the parent resource pool. This can be the root resource pool for a cluster
+  or standalone host, or a resource pool itself. When moving a resource pool
+  from one parent resource pool to another, both must share a common root
+  resource pool or the move will fail.
+* `cpu_share_level` - (Optional) The CPU allocation level. The level is a
+  simplified view of shares. Levels map to a pre-determined set of numeric
+  values for shares. Can be one of `low`, `normal`, `high`, or `custom`. When
+  `low`, `normal`, or `high` are specified values in `cpu_shares` will be
+  ignored.  Default: `normal`
+* `cpu_shares` - (Optional) The number of shares allocated for CPU. Used to
+  determine resource allocation in case of resource contention. If this is set,
+  `cpu_share_level` must be `custom`.
+* `cpu_reservation` - (Optional) Amount of CPU (MHz) that is guaranteed
+  available to the resource pool. Default: `0`
+* `cpu_expandable` - (Optional) Determines if the reservation on a resource
+  pool can grow beyond the specified value if the parent resource pool has
+  unreserved resources. Default: `true`
+* `cpu_limit` - (Optional) The CPU utilization of a resource pool will not exceed
+  this limit, even if there are available resources. Set to `-1` for unlimited.
+  Default: `-1`
+* `memory_share_level` - (Optional) The CPU allocation level. The level is a
+  simplified view of shares. Levels map to a pre-determined set of numeric
+  values for shares. Can be one of `low`, `normal`, `high`, or `custom`. When
+  `low`, `normal`, or `high` are specified values in `memory_shares` will be
+  ignored.  Default: `normal`
+* `memory_shares` - (Optional) The number of shares allocated for CPU. Used to
+  determine resource allocation in case of resource contention. If this is set,
+  `memory_share_level` must be `custom`.
+* `memory_reservation` - (Optional) Amount of CPU (MHz) that is guaranteed
+  available to the resource pool. Default: `0`
+* `memory_expandable` - (Optional) Determines if the reservation on a resource
+  pool can grow beyond the specified value if the parent resource pool has
+  unreserved resources. Default: `true`
+* `memory_limit` - (Optional) The CPU utilization of a resource pool will not exceed
+  this limit, even if there are available resources. Set to `-1` for unlimited.
+  Default: `-1`
+* `tags` - (Optional) The IDs of any tags to attach to this resource. See
+  [here][docs-applying-tags] for a reference on how to apply tags.
+
+[docs-about-morefs]: /docs/providers/vsphere/index.html#use-of-managed-object-references-by-the-vsphere-provider
+[docs-applying-tags]: /docs/providers/vsphere/r/tag.html#using-tags-in-a-supported-resource
+
+## Attribute Reference
+
+The only attribute this resource exports is the `id` of the resource, which is
+the [managed object ID][docs-about-morefs] of the resource pool.
+
+[docs-about-morefs]: /docs/providers/vsphere/index.html#use-of-managed-object-references-by-the-vsphere-provider
```

Comment:
Reference slugs in Markdown don't need to be supplied more than once, this can be omitted.